packages/twenty-front/src/modules/object-metadata/utils/formatFieldMetadataItemInput.ts

```diff
@@ -5,8 +5,19 @@ import { Field } from '~/generated-metadata/graphql';
 
 import { FieldMetadataOption } from '../types/FieldMetadataOption';
 
-const getOptionValueFromLabel = (label: string) =>
-  toSnakeCase(label.trim()).toUpperCase();
+const getOptionValueFromLabel = (label: string) => {
```

Comment:
it would be great to add tests on this function

Reply:
@Weiko Thanks for the tests, I've just added an emoji test